set.mm

```diff
@@ -59996,6 +60040,17 @@ Contrast with domain (defined in ~ df-dm ).  For alternate definitions,
       ZATSBJCJKZDLUAELMZCNBNAUBBCFOBCDEPQR $.
   $}
 
+  ${
+    $d x y A $.  $d x y B $.  $d ph x $.  $d ph y $.
+    eqbrriv2.1 $e |- ( ( ( Rel A /\ Rel B ) /\ ph )
+                        -> ( x A y <-> x B y ) ) $.
+    $( Other version of ~ eqbrrdiv .  (Contributed by Rodolfo Medina,
+       30-Sep-2010.) $)
+    eqbrriv2 $p |- ( ( ( Rel A /\ Rel B ) /\ ph ) -> A = B ) $=
+      ( wrel wa wceq cv wbr cop wcel df-br 3bitr3g eqrelrdv2 anabss5 ) DGEGHZAD
```

Comment:
The `eqbrriv2` naming was suggested by Alexander so I used it. As you point out this is not a well-formed deduction version, I would have written this theorem like this in deduction form:
```
    eqbrriv2.1 $e |- ( ph -> Rel A ) $.
    eqbrriv2.2 $e |- ( ph -> Rel B ) $.
    eqbrriv2.3 $e |- ( ph -> ( x A y <-> x B y ) ) $.
    $( Other version of ~ eqbrrdiv .  (Contributed by Rodolfo Medina, 30-Sep-2010.) $)
    eqbrriv2 $p |- ( ph -> A = B ) $=
```
However, if I rewrite this theorem as such, then I would also have to change Rodolfo's proof of `prter3` in his own Mathbox. We generally try to avoid that.

Reply:
**Edit**
Actually what I'm proposing is exactly the statement of `eqbrrdv`! Then I would prefer _not_ to move `eqbrriv2 ` to the main part of `set.mm` and to ask Peter to rewrite his proofs using `~ eqbrrdv`

Reply:
@tirix the already existing ~eqbrrdv is exactly what you propose! Using this theorem, the proof of ~ eqbrriv2 can be shortened:
       ( wrel wa simpll simplr eqbrrdv ) DGZEGZHAHBCDELMAILMAJFK $.
Knowing this, I think we should NOT move eqbrriv2 to main, but Peter should directly use ~eqbrrdv instead of ~eqbrriv2.

Reply:
I think out messages crossed!
Obviously we agree, I'll merge this with `eqbrriv2` _not_ moved.